stunnel/templates/upstart.jinja

```diff
@@ -12,6 +12,7 @@ console log
 
 expect fork
 exec /usr/bin/stunnel4 {{ stunnel.lookup.conf_dir }}/stunnel.conf
+limit nofile 65536 65536
```

Comment:
Can you make this configurable via pillar please. That should mean just adding a default [here](https://github.com/heap/stunnel-formula/blob/master/stunnel/map.jinja#L4) and referencing it like the `stunnel.lookup.conf_dir` above.